### ISSUE TITLE
Add global cli.args file (configuration file)

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-cli#252]
Zonemaster-cli has the feature that allows the user to create a configuration file (~/.zonemaster/cli.args) with default command line options:
```
CONFIGURATION
       If there is a readable file .zonemaster/cli.args in the user's home directory, each line in that file will be
       prepended as an argument on the command line. For example, if one would like to by default run with the log level
       set to DEBUG and with translation to human-readable messages turned off, one could put this in the config file:

          --raw
          --level=DEBUG

       Any arguments actually given on the command line will override what's in the config file.
```
(from `man zonemaster-cli`)

Add the support for a global configuration file (/etc/zonemaster/cli.args ; /usr/local/etc/zonemaster/cli.args for FreeBSD) that works in the same way.
* Any options in the private configuration file should overwrite options in the global configuration file.
* Any options on the command line should also overwrite.

The new feature was requested in an email to the zonemaster-users list by @aabdnn.

**Comment by matsduf** [comment]
@aabdnn, could you provide a description of your need for this feature as a comment to this issue?

**Comment by mattias-p** [comment]
Oh. I wasn't aware of the discussion on the mailing list.

**Comment by anandb-ripencc** [comment]
My use case is that we have a Zonemaster server, used by multiple users to do checks. Some use the web gui, and some use the cli, but cli users are at a disadvantage, because they would all have to remember that we have a custom profile and use it.

I am aware of `~/.zonemaster/cli.args`, which can be used to specify custom arguments for the cli. However, this is per-user. If this scheme were extended to instead (or additionally) read `/etc/zonemaster/cli.ini` (similar to how the backend reads backend_config.ini), then a custom profile can be defined there globally for all cli users.

**Comment by mattias-p** [comment]
Thank you so much for explaining! I had no idea Zonemaster was being used like this.

**Comment by matsduf** [comment]
Resolved by #260.
